scripts/fly-authelia-provision.test.ts

```
@@ -0,0 +1,264 @@
+        clientSecretHash: '$argon2id$hash',
+        cookieDomain: 'footnote-auth.fly.dev',
+    });
+    assert.match(configuration, /require_pkce: true/);
+    assert.match(configuration, /pkce_challenge_method: 'S256'/);
+    assert.match(
+        configuration,
+        /token_endpoint_auth_method: 'client_secret_basic'/
+    );
+    assert.match(
+        configuration,
+        /scopes:\n          - 'openid'\n          - 'profile'/
```

ESLint

[error] Spaces are hard to count. Use {10}.

(no-regex-spaces)